### ISSUE TITLE
Options in the ConfirmModal are not centered

### ISSUE THREAD

**Issue by xRiku** [thestonechat/habits-sh#1]
The options when deleting a habit are not centered. Should they be, or it was intended? 
![image](https://github.com/user-attachments/assets/1b8b3b12-c394-482d-b59b-2c2878af5916)

**Comment by FranprzDev** [comment]
You mean the container of buttons? or what? I think i can do it

**Comment by xRiku** [comment]
That's right. I could do it too, it would be as simple as changing the justify-end to something else

**Comment by thestonechat** [comment]
this was intentional but it looks awful lol

**Comment by thestonechat** [comment]
fixed!
<img width="320" alt="image" src="https://github.com/user-attachments/assets/ced6bffe-3c49-42c8-aba2-414f206deda3">
